### ISSUE TITLE
Summary to a user submitted story shows up only after multiple submissions

### ISSUE THREAD

**Issue by fareem1** [newsvoice/newsvoice#198]
New story submitted and approved
A summary to the story will not show up on the feed even after approved
Need to submit more than one, sometime multiple, for it to show

Eg. Story : https://newsvoice.com/i/3846584

![Screen Shot 2019-09-05 at 11 59 34](https://user-images.githubusercontent.com/40382741/64332330-a92fa880-cfd4-11e9-98ca-96e395b00bf9.png)

**Comment by vlidholt** [comment]
Fixed.
